src/app.module.ts

```diff
@@ -15,20 +16,10 @@ import { TokenModule } from './token/token.module';
       load: [general],
     }),
     CacheModule.registerAsync({
-      isGlobal: true,
       imports: [ConfigModule],
       inject: [ConfigService],
-      useFactory: async (configService: ConfigService) => {
-        return {
-          store: await redisStore({
-            socket: {
-              host: configService.get<string>('redisHost'),
-              port: configService.get<number>('redisPort'),
-            },
-            password: configService.get<string>('redisPassword'),
-          }),
-        } as unknown as CacheStore;
-      },
+      useFactory: (configService: ConfigService) =>
```

Comment:
@rattrap  For some reason that I couldn't find in the module exposed by the new and old stores global initialization doesn't work as the cacheManager defaults to use in-memory caching in services of submodules...